### PR TITLE
Update native SDK to 15.2.3 for iOS and 15.2.2 for Android

### PR DESCRIPTION
**Pull request by orestesgaolin**
- bumped package version
- updated native SDKs

### CODE REVIEW

**Comment by orestesgaolin** [comment]
Maybe it's safer to bump it to 7.9.0 due to Android target SDK change?

File: intercom_flutter/example/android/app/build.gradle [hunk 1]
```diff
@@ -41,7 +41,7 @@ android {
     defaultConfig {
         applicationId "io.maido.intercomexample"
         minSdkVersion 21
-        targetSdkVersion 33
+        targetSdkVersion 34
```
Comment (deepak786) [inline intercom_flutter/example/android/app/build.gradle]: Need to update the compileSdkVersion also

File: intercom_flutter/README.md [hunk 2]
```diff
@@ -5,9 +5,10 @@
 
 Flutter wrapper for Intercom [Android](https://github.com/intercom/intercom-android), [iOS](https://github.com/intercom/intercom-ios), and [Web](https://developers.intercom.com/installing-intercom/docs/basic-javascript) projects.
 
-- Uses Intercom Android SDK Version `15.2.0`.
+- Uses Intercom Android SDK Version `15.2.2`.
 - The minimum Android SDK `minSdkVersion` required is 21.
-- Uses Intercom iOS SDK Version `15.2.1`.
+- The target Android SDK `targetSdkVersion` required is 34.
```
Comment (deepak786) [inline intercom_flutter/README.md]: compileSdkVersion

**Comment by deepak786** [comment]
> Maybe it's safer to bump it to 7.9.0 due to Android target SDK change?

Lets bump to 7.9.0

File: intercom_flutter/pubspec.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: intercom_flutter
 description: Flutter plugin for Intercom integration. Provides in-app messaging
   and help-center Intercom services
-version: 7.8.5
+version: 7.8.6
```
Comment (alirp88) [inline intercom_flutter/pubspec.yaml]:
```suggestion
version: 7.9.0
```

File: intercom_flutter/example/android/app/build.gradle (hunk 1 above)
Comment (Zazo032) [reply intercom_flutter/example/android/app/build.gradle]: Also `compileSdkVersion` is deprecated, `compileSdk` should be used instead

File: intercom_flutter/example/android/build.gradle [hunk 4]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.7.20'
+    ext.kotlin_version = '1.9.0'
```
Comment (deepak786) [inline intercom_flutter/example/android/build.gradle:2]: It was giving the following error:
```
Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.
```

**Review by Zazo032 — APPROVED** [review]

**Pull request merged**